introduction.md

```diff
@@ -1 +1,6 @@
-Hello Dojo.  My name is Joe and this is an introduction.  My favorite Code School path is Javascript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?